python/cudf/cudf/tests/test_series.py

```diff
@@ -1196,3 +1196,47 @@ def test_explode(data, ignore_index, p_index):
             assert_eq(expect, got, check_dtype=False)
     else:
         assert_eq(expect, got, check_dtype=False)
+
+
+@pytest.mark.parametrize(
+    "arg",
+    [
+        1,
+        -1,
+        "b",
+        np.int32(1),
+        np.uint32(1),
+        np.int8(1),
+        np.uint8(1),
+        np.int16(1),
+        np.uint16(1),
+        np.int64(1),
+        np.uint64(1),
+    ],
+)
+def test_series_indexing(arg):
```

Comment:
Let's move the indexing tests here: https://github.com/rapidsai/cudf/blob/d1c324510a8a0c00de21503d4853a1f88c59b3b2/python/cudf/cudf/tests/test_indexing.py#L102